src/libraries/Common/src/Interop/Windows/Kernel32/Interop.SYSTEM_INFO.cs

```diff
@@ -8,6 +8,7 @@ internal static partial class Interop
 {
     internal static partial class Kernel32
     {
+        [BlittableType]
```

Comment:
This looks like unnecesary ceremony.

Can we allow omitting the `BlittableType` annotations for blittable types from the same assembly?

Reply:
https://github.com/dotnet/runtimelab/issues/1125

Reply:
Removed - @jkoritzinsky added the support to allow omitting `BlittableType` on internal types.